language/stdlib/modules/libra_configs.move

```diff
@@ -31,38 +40,75 @@ module LibraConfig {
         });
     }
 
-    // We can't return reference here, this returns a copy of the config value.
+    public fun apply_for_default_privilege() {
+        if(!Association::addr_is_association(Transaction::sender())) {
+            Association::apply_for_association();
+        };
```

Comment:
This check seems superfluous with the other check that was added in the `Association` module. What do you think about removing that one and keeping this?